pkg/packagekit/updates.jsx

```diff
@@ -284,7 +286,8 @@ function updateItem(info, pkgNames, key) {
         pkgsTruncated.push(<span key="more">…</span>);
 
     if (pkgNames.some(p => p.name.startsWith("kpatch-patch")))
-        pkgsTruncated.push(<React.Fragment key={`${key}-kpatches-spacer`}>{" "}<Badge key={key + "kpatches"}>{_("patches")}</Badge></React.Fragment>);
+        pkgsTruncated.push(<LabelGroup key={`${key}-kpatches-spacer`} className="kpatches-labelgroup">
+            {" "}<Badge color="blue" variant="filled" isCompact>{_("patches")}</Badge></LabelGroup>);
```

Comment:
Not a veto, but perhaps you can expand this to have proper jsx indentation/line breaks.